test/core/list/sort.wren

```diff
@@ -0,0 +1,17 @@
+class A {
+    value { _value }
+
+    toString { _value.toString }
+
+    construct new(value) {
+        _value = value
+    }
+
+    <=(other) { _value <= other.value }
```

Comment:
Just a small observation, but the comparison I've seen used consistently in other languages by sorting algorithms is the "less than" operator and not the "less than or equals" operator. Was there any particular reason why you picked `<=`? I think it may be a little confusing.